addon_imps/storage/dropbox.py

```diff
@@ -0,0 +1,100 @@
+import dataclasses
+import typing
+
+from addon_toolkit.interfaces import storage
+from addon_toolkit.interfaces.storage import ItemType
+
+
+class DropboxStorageImp(storage.StorageAddonHttpRequestorImp):
+    """storage on dropbox.com
+    see https://www.dropbox.com/developers/documentation/http/documentation
+    """
+
+    async def get_external_account_id(self, _: dict[str, str]) -> str:
+        return ""
```

Comment:
Do we not need the external account id? Or does Dropbox not have external account id available?